components/bl-popover-component/src/index.js

```diff
@@ -15,6 +16,14 @@ export default function Popover({ component, eventHandlers, pods }) {
 
   component.setIsOpen = setIsOpen;
 
+  const onMouseEnter = () => {
+    setTimeout(() => onMouseOver({ isOpen }), delayMouseOver);
+  };
+
+  const onMouseLeave = () => {
+    setTimeout(() => onMouseOut({ isOpen }), delayMouseOut);
+  };
```

Comment:
I assume there will be a bug when we make delay for example 5s and spam enter and leave.
We should create and remember timeout, and if new was created but old not yet runned, we should remove old.